benchmarks/user/benchmark.py

```diff
@@ -245,17 +250,17 @@ def plot(problem, **kwargs):
         warning("Could not load any results, nothing to plot. Exiting...")
         sys.exit(0)
 
-    gflopss = bench.lookup(params=kwargs, measure="gflopss", event='main')
-    oi = bench.lookup(params=kwargs, measure="oi", event='main')
-    time = bench.lookup(params=kwargs, measure="timings", event='main')
+    gflopss = bench.lookup(params=kwargs, measure="gflopss", event='section0')
+    oi = bench.lookup(params=kwargs, measure="oi", event='section0')
+    time = bench.lookup(params=kwargs, measure="timings", event='section0')
```

Comment:
Well this is not gonna be correct for tti staggered for example that is split into two major loops + extras. I am not sure what should be taken but taking one section by name seems dangerous.

Reply:
I agree absolutely with that, may be we can discuss tomorrow.

Reply:
this is hacky, yes, but it can't be worse than not working

also, I **thought** staggered tti, just like all other examples supported by benchmark.py, actually ended up putting all FD stencils in section0

Reply:
Should we add a comment or a note in a README.md about that?

Reply:
we should aim for a fix...maybe not this PR, but that's what we should do. Maybe a hacky one! It's just a benchmarking script after all.

Maybe we could let users attach a name to Eqs, to be used in place of numbers in the section name?
```
Eq(...., name=X)
```
and then in the generated code
```
/* Begin section X */
...
....
```

though this is probably future work